packages/core/src/httpClient/retryableHttpClient.ts

```diff
@@ -15,14 +20,71 @@ export class RetryableHttpClient extends BasicHttpClient {
     super(baseUrl);
   }
 
+  public get<ResponseType>(
+    path: string,
+    options: RetryableHttpRequestOptions = {}
+  ) {
+    return super.get<ResponseType>(path, options);
+  }
+
+  public post<ResponseType>(
+    path: string,
+    options: RetryableHttpRequestOptions = {}
+  ) {
+    return super.post<ResponseType>(path, options);
+  }
+
+  public put<ResponseType>(
+    path: string,
+    options: RetryableHttpRequestOptions = {}
+  ) {
+    return super.put<ResponseType>(path, options);
+  }
+
+  public delete<ResponseType>(
+    path: string,
+    options: RetryableHttpRequestOptions = {}
+  ) {
+    return super.delete<ResponseType>(path, options);
+  }
+
+  public patch<ResponseType>(
+    path: string,
+    options: RetryableHttpRequestOptions = {}
+  ) {
+    return super.patch<ResponseType>(path, options);
+  }
+
+  protected async preRequest(
+    request: RetryableHttpRequest
+  ): Promise<RetryableHttpRequest> {
+    return super.preRequest(request);
+  }
+
+  protected async postRequest<T>(
+    response: HttpResponse<T>,
+    _: RetryableHttpRequest // eslint-disable-line
```

Comment:
What's the problem here? And why not 
```suggestion
    request: RetryableHttpRequest
```

Reply:
I copied it from `BasicHttpClient`, but no reason to keep it that way 👍🏻